src/main/java/com/scalar/db/benchmarks/tpcc/TpccLoader.java

```diff
@@ -161,27 +108,134 @@ static class Exclusive {
           .put(STOCK, STOCK_HEADER)
           .put(WAREHOUSE, WAREHOUSE_HEADER)
           .build();
-  private int endWarehouse;
+  private final DistributedTransactionManager manager;
+  private final int concurrency;
+  private final int startWarehouse;
+  private final int endWarehouse;
+  private final boolean skipItemLoad;
+  private final boolean useTableIndex;
+  private String directory;
+
+  public TpccLoader(Config config) {
+    super(config);
+    DatabaseConfig dbConfig = getDatabaseConfig(config);
+    TransactionFactory factory = new TransactionFactory(dbConfig);
+    manager = factory.getTransactionManager();
+    manager.withNamespace(TpccRecord.NAMESPACE);
 
-  public static void main(String[] args) {
-    int exitCode = new CommandLine(new TpccLoader()).execute(args);
-    System.exit(exitCode);
+    this.concurrency =
+        (int) config.getUserLong(CONFIG_NAME, LOAD_CONCURRENCY, DEFAULT_LOAD_CONCURRENCY);
+    this.skipItemLoad = config.getUserBoolean(CONFIG_NAME, SKIP_ITEM_LOAD, DEFAULT_SKIP_ITEM_LOAD);
+    this.useTableIndex =
+        config.getUserBoolean(CONFIG_NAME, USE_TABLE_INDEX, DEFAULT_USE_TABLE_INDEX);
+    try {
+      this.directory = config.getUserString(CONFIG_NAME, CSV_FILE_DIRECTORY);
+    } catch (IllegalConfigException e) {
+      this.directory = null;
```

Comment:
Actually, this was neither a warning nor an error since it just happens when users don't specify the directory. So, I fixed to pre-check it so that the exception is not thrown here. Fixed in 293b02b.